funcs/func_global.c

```diff
@@ -53,6 +53,19 @@
 			<para>Set or get the value of a global variable specified in <replaceable>varname</replaceable></para>
 		</description>
 	</function>
+	<function name="GLOBAL_DELETE" language="en_US">
+		<synopsis>
+			Deletes a specified global variable.
+		</synopsis>
+		<syntax>
+			<parameter name="varname" required="true">
+				<para>Global variable name</para>
+			</parameter>
+		</syntax>
+		<description>
+			<para>Delete the global variable specified in <replaceable>varname</replaceable></para>
```

Comment:
To use the same verb tense as other descriptions, I think "Deletes" would be better than "Delete"
Also might be good to state that it will work (always return 0) regardless of whether the variable existed or not, I think some people might wonder about that.

Reply:
I am amused as the verb tense matches that of the existing dialplan function this was copied from.

Reply:
> I am amused as the verb tense matches that of the existing dialplan function this was copied from.

Hmm, I was looking at the Gets/Sets nearby... in that case just leave it as is.